### PR TITLE
SWTCH-2223 update dependencies

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "@energyweb/credential-governance": "^2.2.1-alpha.316.0",
     "@energyweb/vc-verification": "^2.2.1-alpha.316.0",
-    "@ensdomains/ens": "^0.4.5",
```
Comment (JGiter) [inline package.json:57]: @ensdomains/ens has been deprecated https://www.npmjs.com/package/@ensdomains/ens

File: package.json [hunk 2]
```diff
@@ -163,7 +163,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-no-only-tests": "^2.6.0",
-    "ethereum-waffle": "^3.4.4",
```
Comment (JGiter) [inline package.json:166]: `ethereum-waffle@3.4.4` depends on deprecated `@ensdomains/ens`. Although it is not runtime dependency it provides a bit more confidency in tests

File: package.json [hunk 3]
```diff
@@ -122,8 +122,7 @@
     "siwe": "^1.1.6",
     "swagger-ui-express": "^4.1.4",
     "typechain": "^8.1.1",
-    "typeorm": "^0.3.6",
-    "useragent": "^2.3.0",
```
Comment (JGiter) [inline package.json:126]: `useragent` is not used

File: src/modules/did/pin.processor.ts [hunk 4]
```diff
@@ -75,7 +75,9 @@ export class PinProcessor {
   @Process(PIN_CLAIM_JOB_NAME)
   async pinClaims(doc: DIDDocumentEntity) {
     for (const cid of doc.service.map((s) => s.serviceEndpoint)) {
-      if (!(await this.didCluster.isPinned(cid))) {
```
Comment (JGiter) [inline src/modules/did/pin.processor.ts:78]: cluster specific methods were removed from DidStore

File: package.json [hunk 5]
```diff
@@ -76,12 +76,12 @@
     "@nestjs/platform-express": "^9.2.1",
     "@nestjs/schedule": "^2.1.0",
     "@nestjs/swagger": "^6.1.3",
-    "@nestjs/terminus": "^9.1.4",
```
Comment (JGiter) [inline package.json:79]: updated because depends on `typeorm`

**Review by nichonien — APPROVED** [review]

**Pull request merged**